pkg/clientinteractor/interactor.go

```diff
@@ -383,7 +383,7 @@ type ClientDesc struct {
 
 func (_ ClientDesc) GetRow(cl client.Client, hostname string, rAddr string) []string {
 	quantiles := statistics.GetQuantiles()
-	rowData := []string{cl.ID(), cl.Usr(), cl.DB(), hostname, rAddr}
+	rowData := []string{fmt.Sprintf("%d", cl.ID()), cl.Usr(), cl.DB(), hostname, rAddr}
```

Comment:
This is slow

Reply:
We cant do much here, plus this is not hot code path